core/node/tee_verifier_input_producer/src/lib.rs

```diff
@@ -6,21 +6,20 @@
 //! For testing purposes, the L1 batch is re-executed immediately for now.
 //! Eventually, this component will only extract the inputs and send them to another
 //! machine over a "to be defined" channel, e.g., save them to an object store.
+#![feature(slice_flatten)]
```

Comment:
No nightly features are allowed in the core, we're only using nightly compiler because we are forced by our dependencies.

Reply:
so `get_factory_deps()` returns the bytecode in chunks of `Vec<[u8; 32]>`, but `pub fn store_factory_dep(&mut self, hash: H256, bytecode: Vec<u8>)` in `InMemoryStorage` wants flat byte vecs... maybe bytecode should get its own type... also the conversion between U256 and H256 all the time is really annoying.

Reply:
That's true, there are many places that can be improved in the codebase, it's mostly a question of having (no) resources to work on low-priority stuff. Not that it's not important, more that it works and we've got plenty of other areas where there are "hotter" issues.

Reply:
"fixed"